src/App.vue

```diff
@@ -222,6 +222,7 @@ Sentry.init({
     min-height: 0;  /* don't expand to children content -- undo auto min-height */
     max-height: 100%;
     max-width: 600px;
+    padding-right: var(--sz-100);
```

Comment:
Malheureusement quand j'essaye localement sur 320x450 ça push le "Évènements extrêmes" sur deux ligne ça apparemment:
![image](https://user-images.githubusercontent.com/1843555/190872617-ae8066c4-4516-43b3-9168-5aa93e53e67d.png)

Mais c'est ok tu peux commit as-is, peut-être qu'on peut follow-up en réduisant le size du tag de température courante pour être un peu plus petit (comme sur le design) et éviter d'avoir besoin de pad-right.

Reply:
À noter que 320x450 y'a aucun device qui a cette résolution là. Le iPhone 5 a "techniquement" 320x568, mais c'est un display retina, sa résolution réelle est le double de ça.